packages/emotion/test/test_css_js_tag.ml

```diff
@@ -0,0 +1,345 @@
+let assert_string left right =
+  Alcotest.check Alcotest.string "should be equal" right left
+
+let assert_not_equal_string left right =
+  Alcotest.check (Alcotest.neg Alcotest.string) "should not be equal" right left
+
+let render_style_html_tag () =
+  let content = CssJs.render_style_html_tag () in
+  let _ = CssJs.flush () in
+  content
+
+let one_property () =
+  let className = CssJs.style [| CssJs.display `block |] in
+  let css = render_style_html_tag () in
+  assert_string css
+    (Printf.sprintf
+       "<style data-s data-emotion=\"css %s\">.%s { display: block; }</style>"
+       className className)
+
+let multiple_properties () =
+  let className =
+    CssJs.style [| CssJs.display `block; CssJs.fontSize (`px 10) |]
+  in
+  let css = render_style_html_tag () in
+  assert_string css
+    (Printf.sprintf
+       "<style data-s data-emotion=\"css %s\">.%s { display: block; font-size: \
+        10px; }</style>"
```

Comment:
We might not need the same tests cases as test_css_js_styles and we can live with one simple case and a "real world" example.